### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by karanJ2212**
In this milestone I have implemented following requirements-

> Implement the design from the sneak peek wireframes.
> Styling is nice to have but focus on making sure everything functions first.
> Use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).

### CODE REVIEW

File: app/models/comment.rb [hunk 1]
```diff
@@ -1,11 +1,9 @@
 class Comment < ApplicationRecord
-  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
-  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+  belongs_to :user, foreign_key: 'user_id'
+  belongs_to :post
```
Comment (DeliceLydia) [inline app/models/comment.rb]: - [ ] For some reason I am not able to create a comment I suggest that you check your association and foreign_keys
Screen Shot 2023-06-29 at 18 17 29

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @karanJ2212, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The app is rendering the user's 3 most recent posts
- No linter errors
- Good README


## Required Changes ♻️
_Check the comments under the review._



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/comment.rb (hunk 1 above)
Comment (karanJ2212) [reply app/models/comment.rb]: resolved

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @karanJ2212 👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job on implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉

## Highlights
✔ Added the Views associated with the Models and Controllers .👍
✔ Made suggested changes👍
✔ Descriptive PR👍

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/enus/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**